Project/DAQ_System/Core/Inc/stm32f4xx_it.h

```diff
@@ -52,8 +52,6 @@ void MemManage_Handler(void);
 void BusFault_Handler(void);
 void UsageFault_Handler(void);
 void DebugMon_Handler(void);
-void CAN1_RX0_IRQHandler(void);
-void CAN1_RX1_IRQHandler(void);
```

Comment:
Are interrupts for receiving CAN messages from the ECU now disabled?

Reply:
They seem to be on this version, I think I might've misclicked or something on CubeMx, fixed